local-cli/server/findSymlinksPaths.js

```diff
@@ -6,7 +6,28 @@ module.exports = function findSymlinksPaths(lookupFolder) {
   const folders = fs.readdirSync(lookupFolder);
   const resolvedSymlinks = folders.map(folder => path.resolve(lookupFolder, folder))
     .filter(folderPath => fs.lstatSync(folderPath).isSymbolicLink())
-    .map(symlink => path.resolve(process.cwd(), fs.readlinkSync(symlink)));
+    .map(symlink => {
+      const symlinkCache = new Set();
+      let prevSymlink;
+      let nextSymlink = symlink;
+      while (true) {
+        if (symlinkCache.has(nextSymlink)) {
+          throw Error(`Symlink recursion detected:\n  ${nextSymlink}`);
```

Comment:
Will be really great if we can throw an error with the whole sequence that leads to the recursion. Like "Symlink recursion detected: A -> B -> C -> B"